src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -44,6 +44,10 @@ public abstract class FormulaType
         public static FormulaType Color { get; } = new ColorType();
 
         public static FormulaType Guid { get; } = new GuidType();
+
+        public static FormulaType Unknown { get; } = new UnknownType();
+
+        public static FormulaType Error { get; } = new ErrorType();
```

Comment:
People may abuse this - can we make safer.

Reply:
Remove these completely perhaps? Then there is no public way anymore to create those two types (ctors are internal now).

Reply:
Or at least do this for error type and leave for unknown?